src/utils/nanvixd/src/logging.rs

```diff
@@ -25,7 +25,7 @@ use ::std::sync::Once;
 pub fn initialize() {
     static INIT_LOG: Once = Once::new();
     INIT_LOG.call_once(|| {
-        Logger::try_with_env()
+        Logger::try_with_env_or_str("error")
```

Comment:
Is this intentional?

Reply:
Yes, it got squashed with all the other changes, but doing this ensures that ERROR messages are printed if you don't run with any `RUST_LOG` env. var set.

Otherwise, even if the code fails you will see no error messages. If you want this to be the behaviour I can revert the change.

Reply:
Oh I see... I liked it, but we may want to fix that in the other projects (ie microvm and linuxd). Can you create a good first issue for that?

Reply:
Done: https://github.com/nanvix/nanvix/issues/704

Do you want me to undo these changes and apply them all in a separate PR?

Reply:
Yes :)